charts/sn-platform/values.yaml

```diff
@@ -600,6 +600,40 @@ zookeeper:
     # requiredDuringSchedulingIgnoredDuringExecution - rules must be met for pod to be scheduled (hard) requires at least one node per replica
     # preferredDuringSchedulingIgnoredDuringExecution - scheduler will try to enforce but not guranentee
     type: preferredDuringSchedulingIgnoredDuringExecution
+
```

Comment:
@FushuWang @fantapsody Can we just provide a custom affinity roles config like 

```
affinity:
    anti_affinity: true
    type: preferredDuringSchedulingIgnoredDuringExecution

    customAffinity: {}
```

to override the default rule of `spec.affinity`? No matter users want to set `podAffinity`,`nodeAffinity` or even mixed affinity rules.

Reply:
For example like 

```
affinity:
    anti_affinity: true
    type: preferredDuringSchedulingIgnoredDuringExecution

    customAffinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: kubernetes.io/os
                operator: In
                values:
                - linux
          preferredDuringSchedulingIgnoredDuringExecution:
          - weight: 1
            preference:
              matchExpressions:
              - key: another-node-label-key
                operator: In
                values:
                - another-node-label-value
        podAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
          - labelSelector:
              matchExpressions:
              - key: security
                operator: In
                values:
                - S1
            topologyKey: topology.kubernetes.io/zone
```

Reply:
@ericsyh Yeah, I think we should allow users to set these rules freely as needed.